### ISSUE TITLE
overall ranking

### ISSUE THREAD

**Issue by xxrelax** [EDAPINENUT/CBGBench#10]
Dear author,
I have a question regarding Table 5: Ranking scores and overall ranking in your paper.  Specifically, I would like to confirm if the final rankings were obtained through a weighted average of the four scores, followed by ranking the results.

Thank you very much for your time and for sharing your valuable work with the community.

**Comment by EDAPINENUT** [comment]
Yes, it is a weighted calculation of the ranking scores, which each kind of score is calculated by  weight * (12 - rank)

**Comment by xxrelax** [comment]
Thank you for your reply
